cli/config/compose/services/elastic-agent-ubi/docker-compose.yml

```
@@ -0,0 +1,14 @@
+version: '2.3'
+services:
+  elastic-agent-ubi:
+    image: docker.elastic.co/beats/elastic-agent-ubi8:${elasticAgentTag:-8.0.0-SNAPSHOT}
```

Could we reuse the same scenario as for "elastic-agent"? Maybe somehow parameterizing the suffix here.
```suggestion
    image: docker.elastic.co/beats/elastic-agent${elasticAgentDockerImageSuffix}:${elasticAgentTag:-8.0.0-SNAPSHOT}
```

---

Yes, we can do this. I like it!